### ISSUE TITLE
"Geo range search" - the coordinates in the property tiles can not be hidden

### ISSUE THREAD

**Issue by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#539]
### Discussed in https://github.com/onOffice-Web-Org/oo-wp-plugin/discussions/534

<div type='discussions-op-text'>

<sup>Originally posted by **fredericalpers** May 26, 2023</sup>
### Current state

Once the "geoPosition" field is added for "geo range search", the property tiles always output latitude & longitude, whether you have it set to "invisible" or not. See screenshot attached.

### Current workaround:

$dontEcho = array("object title", / *...* / "longitude", "latitude");

### Desired state
If the field is set to "invisible", it should not be visible in the property tiles.

![geoPosition](https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/88546396/c557e92c-d408-4676-9f27-59fd318f10cd)
![breitengrad-laengengrad](https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/88546396/175ba6f9-3dd4-49d3-95b5-0ef99b0bd697)

</div>

**Comment by fredericalpers** [comment]
@dai-eastgate sorry I got confused with the Filterable and hidden fields myself. (hence we are renaming it #517) 

This issue was not explained clearly enough by me. 

If the geoposition field is added to the fieldlist, the latidude and longitude fields are always shown in the property tiles. In most cases clients do want to add the geo range search (geoposition) so they can use it in the actualy property search. But they do not want it to show in the property tiles. Right now there it always has to be taken out from the property tiles with a template customization.

**Comment by dai-eastgate** [comment]
@fredericalpers 
I have checked in the code and see that 2 fields "breitengrad" and "laengengrad" are added at this commit https://github.com/onOffice-Web-Org/oo-wp-plugin/commit/35473b30224e3c26bf5f386d24bee4216e7d0482?diff=unified. 
That means every time we config the geo position field, 2 fields "breitengrad" and "laengengrad" are displayed in Frontend of the Estate List. Therefore, I would like to confirm whether you want me to delete these 2 fields. Please check back and let me know your opinion about this. Thanks!

![image](https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/106214469/208b0e0c-b114-49ad-9f7a-8ca8551ad550)

**Comment by fredericalpers** [comment]
> @fredericalpers I have checked in the code and see that 2 fields "breitengrad" and "laengengrad" are added at this commit [35473b3?diff=unified](https://github.com/onOffice-Web-Org/oo-wp-plugin/commit/35473b30224e3c26bf5f386d24bee4216e7d0482?diff=unified). That means every time we config the geo position field, 2 fields "breitengrad" and "laengengrad" are displayed in Frontend of the Estate List. Therefore, I would like to confirm whether you want me to delete these 2 fields. Please check back and let me know your opinion about this. Thanks!
> 
> ![image](https://user-images.githubusercontent.com/106214469/246380855-208b0e0c-b114-49ad-9f7a-8ca8551ad550.png)

yes, please delete those. If customers want to display the fields longitude and latitude, they can still add them with the associated fields from the fieldliste.

**Comment by dai-eastgate** [comment]
Yes I will implement it. Thanks
